gleam/src/erl.rs

```diff
@@ -1674,6 +1674,18 @@ x() ->
     'try':'and'().
 "#,
         },
+        // Translation of Float-specific BinOp into variable-type Erlang term comparison.
+        Case {
+            src: r#"fn x() { 1. <. 2.3 }"#,
+            erl: r#"-module().
+-compile(no_auto_import).
+
+-export([]).
```

Comment:
I think this will fail now because of the change to exports, could you rebase on master? Thank you